arches/app/datatypes/base.py

```diff
@@ -462,3 +462,9 @@ def pre_structure_tile_data(self, tile, nodeid, **kwargs):
         Adds properties to a tile necessary for some clients, but not essential to the tile
         """
         pass
+
+    def validate_node(self, node):
+        """
+        Confirms that a node is properly configured to collect data
+        """
+        return {"success": True}
```

Comment:
That would be cleaner, but this is JSON because we need to send a value on the 'message' property to `GraphValidationError`.  All other datatypes are using this to send back success so that the `GraphValidationError` isn't thrown because we aren't actually validating their config in _validate_node_config. We could add some conditional logic in `_validate_node_config` to sometimes accept a boolean and sometimes JSON, but that seems a bit messy.

Reply:
Hmm, I still don't love the idea. How do you feel about throwing an exception ( generic or creating a DatatypeException ) , and wrapping the callsite in a try/catch to harvest the message? That would still avoid this returning JSON but also allow an error message to be passed to the parent.